.github/workflows/zig-build.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow uses Zig and its excellent cross-compilation support to test
+# compiling for multiple platforms. No tests are actually run since it would
+# require emulation. We should add a test for the native platform though.
+name: zig build
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: ${{ matrix.ttriple }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ttriple: [ aarch64-linux-musl, wasm32-wasi, x86_64-linux-gnu.2.27, x86_64-linux-musl, x86_64-windows-gnu ]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: "Install zig"
+      run: |
+        curl -o zig.tar.xz https://ziglang.org/builds/zig-linux-x86_64-0.12.0-dev.1814+5c0d58b71.tar.xz
```

Comment:
Can we do w/o an intermediate (zig.tar.xz) file? Like this (mind '-' symbol):
```
mkdir zig
wget -O - <url> | tar xf - --xz --strip-components=1 --directory zig
```

Reply:
Yes, fixed!